stable-diffusion/requirements.txt

```diff
@@ -0,0 +1,21 @@
+# --------- pytorch --------- #
+torch>=1.12.0
+torchvision>=0.11.0
+mosaicml[all]==0.12.0
+
+
+# --------- loggers --------- #
+wandb
+# neptune-client
+# mlflow
+# comet-ml
+# torch_tb_profiler
```

Comment:
if these are in here for the user's convenience, we should probably prefix them with a comment to that effect so it doesn't get mistaken for unintentional code